v22.1/aws-dms.md

```diff
@@ -0,0 +1,140 @@
+1. You can test the connection if needed under **Test endpoint connection (optional)**.
+1. Create the endpoint by selecting **Create endpoint**.
+    <img src="{{ 'images/v22.1/aws-dms-test-endpoint.png' | relative_url }}" alt="AWS-DMS-Test-Endpoint" style="max-width:100%" />
+
+## Step 2. Create a database migration task
+
+A database migration task, also known as a replication task, controls what data are moved from the source database to the target database.
+
+### Step 2.1. Task configuration
+
+1. While in **AWS DMS**, select **Database migration tasks** in the sidebar. A list of database migration tasks should show, if any exist.
+1. In the top-right portion of the window, select **Create task**.
+    <img src="{{ 'images/v22.1/aws-dms-create-db-migration-task.png' | relative_url }}" alt="AWS-DMS-Create-DB-Migration-Task" style="max-width:100%" />
+1. Supply a **Task identifier** to identify the replication task.
+1. Select the **Replication instance** and **Source database endpoint** you created prior to starting this tutorial.
+1. For the **Target database endpoint** dropdown, select the CockroachDB endpoint created in the previous section.
+1. Select the appropriate **Migration type** based on your needs.
+    <img src="{{ 'images/v22.1/aws-dms-task-configuration.png' | relative_url }}" alt="AWS-DMS-Task-Configuration" style="max-width:100%" />
+
+### Step 2.2. Task settings
+
+1. For the **Editing mode** radio button, keep **Wizard** selected.
+1. For the **Target table preparation mode**, select either **Truncate** or **Do nothing**. All other settings can remain as-is.
+    <img src="{{ 'images/v22.1/aws-dms-task-settings.png' | relative_url }}" alt="AWS-DMS-Task-Settings" style="max-width:100%" />
+
+In order to use **Drop tables on target**, you must [create a new `dms` user](#create-a-new-dms-user) and ensure [`BatchApplyEnabled`](#batchapplyenabled) is set to `False`.
+
+### Step 2.3. Table mappings
+
+1. For the **Editing mode** radio button, keep **Wizard** selected.
+1. Select **Add new selection rule**.
+1. Supply the appropriate **Schema** (if applicable), **Source name** (database name), **Table name**, and **Action**.
+    <img src="{{ 'images/v22.1/aws-dms-table-mappings.png' | relative_url }}" alt="AWS-DMS-Table-Mappings" style="max-width:100%" />
+
+## Step 3. Verify the migration
+
+Data should now be moving from source to target. You can analyze the **Table Statistics** page for information about replication.
+
+1. In **AWS DMS**, open **Database migration tasks** in the sidebar.
+1. Select the task you created in Step 2.
+1. Select **Table statistics** below the **Summary** section.
+
+If your migration failed for some reason, you can check the checkbox next to the table(s) you wish to re-migrate and select **Reload table data**.
+
+## Optional configurations
+
+### Create a new `dms` user
```

Comment:
cc @ajstorm i'm ok with publishing this for now, but i know you're more against it. if i understand, the preference is to *not* select drop tables on target & force the user to put the new schema on crdb first